projects/discussion-ui/src/lib/components/lib-entry/lib-entry.component.html

```diff
@@ -1,3 +1,16 @@
+<div class="relative position">
+    <div class="ui container">
+      <div class="py-8 d-flex flex-ai-center">
+        <!-- /* Back button */ -->
+        <button type="button" class="sb-btn sb-btn-normal sb-btn-link sb-btn-link-primary sb-left-icon-btn mr-auto">
+          <i class="icon-svg icon-svg--xxs icon-back mr-4"><svg class="icon icon-svg--primary">
+              <use xlink:href="assets/images/sprite.svg#arrow-long-left"></use>
```

Comment:
This path is wrong. It should be like below
`./assets/discussion-ui/images/`
Check in other places & do the same